### DISCUSSION TITLE
Using different languages eg (Vlang, Golang)

### DISCUSSION THREAD

**Discussion by M1ndo** [geeksblabla/blanat#236] [Q&A]
I want to write my solution in either V or Go, but they're not mentioned in the README.
Wondering if it could be a submitted as a valid submission?

**Comment by aboullaite** [comment]
Golang is supported as of this pr https://github.com/geeksblabla/blarun/pull/6

**Reply by M1ndo** [reply]
Wonderful, I've also added a pr for Vlang.
